fog/view/connection/src/lib.rs

```diff
@@ -70,10 +95,35 @@ impl FogViewConnection for FogViewGrpcClient {
 
             let aad_bytes = mc_util_serial::encode(&req_aad);
 
-            retry(Fixed::from_millis(100).take(5).map(jitter), || {
-                self.conn
-                    .retriable_encrypted_enclave_request(&req, &aad_bytes)
-            })
+            let retry_config = self.grpc_retry_config;
+            retry_config
+                .retry(|| {
+                    self.conn
+                        .retriable_encrypted_enclave_request(&req, &aad_bytes)
+                })
+                .map_err(|err| Error {
+                    uri: self.uri.clone(),
+                    error: err,
+                })
```

Comment:
Optional/Equivalent: Rename the closure param for succinctness.
```suggestion
                .map_err(|error| Error {
                    uri: self.uri.clone(),
                    error,
                })
```